Zigbee2MqttAssistant/Services/BridgeStateService.cs

```diff
@@ -81,9 +82,43 @@ Bridge Update(Bridge state)
 			return device;
 		}
 
+		private bool IsValidJson(string strInput)
+		{
+			strInput = strInput.Trim();
+			if ((!strInput.StartsWith("{") || !strInput.EndsWith("}")) &&
+			    (!strInput.StartsWith("[") || !strInput.EndsWith("]")))
+			{
+				//basic criteria for a JSON string/object not fulfilled
+				_logger.LogDebug($"Basic criteria for a JSON object is not met '{strInput}'");
+				return false;
+			}
+
+			try
+			{
+				JToken.Parse(strInput);
```

Comment:
You're parsing the string twice. You should output the `JToken` as a `out` parameter in this case and use it by the caller.